### PR TITLE
Add DCAP runtime base Dockerfile

### PR DESCRIPTION
**Pull request by nick-mobilecoin**
Adds a runtime base Dockerfile to be used in DCAP deployments.

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3514** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3514" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3514?utm_source=stack-comment).

File: .internal-ci/docker/Dockerfile.dcap-runtime-base [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2023 The MobileCoin Foundation
+#
+# Dockerfile.runtime-base
```
Comment (jgreat) [inline .internal-ci/docker/Dockerfile.dcap-runtime-base:3]: nit: fix the name here.

File: .internal-ci/docker/Dockerfile.dcap-runtime-base [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+#  A minimal base runtime image for MobileCoin applications.
+#
+FROM ubuntu:focal-20221019
```
Comment (jgreat) [inline .internal-ci/docker/Dockerfile.dcap-runtime-base:6]: lets update this base image.  20230801

File: .internal-ci/docker/Dockerfile.dcap-runtime-base [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  && apt-get clean \
+  && rm -r /var/lib/apt/lists
+
+# The config installed with `libsgx-dcap-default-qpl` is for a local PCCS service.
+# We copy the Azure PCCS config after installing `libsgx-dcap-default-qpl` to
+# override this default config.
+COPY .internal-ci/docker/support/sgx_default_qcnl_azure.conf /etc/sgx_default_qcnl.conf
+
+# Install GRPC health probe
+ARG GRPC_HEALTH_UTILITY_URL=https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.4.7/grpc_health_probe-linux-amd64
```
Comment (jgreat) [inline .internal-ci/docker/Dockerfile.dcap-runtime-base:62]: lets bump this version to v0.4.19

**Review by jgreat — APPROVED** [review]

File: .internal-ci/docker/Dockerfile.dcap-runtime-base (hunk 1 above)
Comment (nick-mobilecoin) [reply .internal-ci/docker/Dockerfile.dcap-runtime-base]: thanks!, fixed

File: .internal-ci/docker/Dockerfile.dcap-runtime-base (hunk 2 above)
Comment (nick-mobilecoin) [reply .internal-ci/docker/Dockerfile.dcap-runtime-base]: updated

File: .internal-ci/docker/Dockerfile.dcap-runtime-base (hunk 3 above)
Comment (nick-mobilecoin) [reply .internal-ci/docker/Dockerfile.dcap-runtime-base]: updated

**Pull request merged**